src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
+type Color = 'red' | 'green' | 'blue';
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => number;
+}
 
+function findInvalidValue(...args: number[]): boolean {
+  return args.some((value) => value <= 0);
 }
 
-export class Triangle {
+export class Rectangle implements Figure {
+  public shape: Shape = Shape.Rectangle;
```

Comment:
Great! Although I don't think you need to put 'public' in front of each variable, they are public by default

Reply:
agree, remove redundant modificator